examples/image_classification/setup.sh

```diff
@@ -0,0 +1,27 @@
+
+base_env=$(conda info | grep -i 'base environment' | awk -F': ' '{print $2}' | sed 's/ (read only)//' | tr -d ' ')
+current_dir=$(pwd)
+
+task=${1}
+env_name=$task
+echo "# ========================================================= #"
+echo "create env for task ${task}.."
+
+if conda env list | grep -q -E "^$env_name\s"; then
+    source ${base_env}/etc/profile.d/conda.sh
+    conda activate ${env_name}
+else
+    conda clean --all --force-pkgs-dir -y
+    conda create --name ${env_name} python=3.8 -y
+    source ${base_env}/etc/profile.d/conda.sh
+    conda activate ${env_name}
+    install_requirements=1
+fi
+echo "environment name: ${env_name}"
+
+if [ "$CONDA_DEFAULT_ENV" = "${env_name}" ] && [ "$install_requirements" == "1" ]; then
+    echo "installing requirements in conda env ${env_name}.."
+    cd ${current_dir}
+    pip install -r requirements2.txt
```

Comment:
Before pip installing, `update-moreh --target 23.x.x --force` should be required

Reply:
There is no file named `requirements2.txt` tho? @kimmorehvn

Reply:
Agree